### ISSUE TITLE
Newest release blocks races

### ISSUE THREAD

**Issue by ghost** [yrmvgh/yiufcrawl#6]
Windows tiles version 1.5.2b1-yiuf-4gc2bcc00e39

Newest release, lacking any race alphabetically past octopode:
![image](http://image.prntscr.com/image/81745b56e8984c8eb3abb1d1d8b37965.png)

**Comment by yrmvgh** [comment]
I rushed a fix to the servers (crawl.project357.org and www.gnollcrawl.tk) for this issue, but haven't rolled a new release for it yet. If I fix the bearkin mutation issue, **OR** if trunk finishes renaming "Bultungins" to "Gnolls", I'll push a release. Here's a quick fix with no other recent changes (which is to say that issue #5 is still unfixed in this file):
[yiuf_soup-1.5.2b1-yiuf-18-g88e35686ef-tiles-win32.zip](https://github.com/yrmvgh/yiufcrawl/files/916169/yiuf_soup-1.5.2b1-yiuf-18-g88e35686ef-tiles-win32.zip)
I edited your comment to show the image directly rather than linking to the image. Edit your comment if you wish to see the syntax in action!
